db/migrate/20230206112201_change_qunatity_from_foods.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeQunatityFromFoods < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :foods, :qunatity, :quantity
+    #Ex:- rename_column("admin_users", "pasword","hashed_pasword")

```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.